tests/interfaces/pygame_adapter/ui/test_command_column.py

```
@@ -0,0 +1,63 @@
+
+        # Check that draw_rect was called with the background color
+        mock_draw_rect.assert_called_once_with(
+            mock_surface, colors.COMMAND_COLUMN_BACKGROUND, self.test_rect
+        )
+
+    @patch("pygame.font.SysFont")
+    def test_font_initialization(self, mock_sys_font: MagicMock) -> None:
+        """Test that fonts are properly initialized."""
+        # Reset the command column to trigger font initialization
+        test_rect = pygame.Rect(0, 0, 200, 600)
+        command_column = CommandColumn(test_rect)
```

Ruff (0.8.2)

Local variable `command_column` is assigned to but never used

Remove assignment to unused variable `command_column`

(F841)